Makefile

```diff
@@ -39,7 +39,7 @@ TAG=$(GIT_USER)-latest
 
 # Preflight Check Defaults
 USE_DOCKER=1
-DOCKER_OUT=$(shell docker ps)
+RUNNING_DOCKER=$(shell docker ps &>/dev/null; echo $$?)
```

Comment:
might be nice if you could invert this value. But likely tricky in `make`... Thinking of something like:
```
RUNNING_DOCKER=1 - $(shell docker ps &>/dev/null; echo $$?)  # probably wrong syntax

```
Then below both vars can be tested for 1...

Reply:
That seems like it would be confusing.  If we doing the command in line, it would be 

`$ if [ $(docker ps &>/dev/null; echo $?) ]; then echo is-true; fi`
If running docker, the exit code would be 0 and it would print `is-true`.